go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/operator-framework/api v0.3.20
 	github.com/operator-framework/operator-lib v0.2.0
 	github.com/spf13/cobra v1.0.0
+	gomodules.xyz/jsonpatch/v2 v2.0.1
```

Comment:
- [ ] @kwiesmueller check this import, looks wrong :thinking:

Reply:
This is what kubernetes uses: https://github.com/evanphx/json-patch
We should probably too.

Reply:
Hm... I looked at the library and the one k8s uses wouldn't help really.
I only use jsonpatch for creating a test patch as having Go types is nicer, but can do without if anybody has concerns with having this dep in here.
Switching would not make a lot of sense, then I'd rather just build the patches with fmt.Sprint or use own types.

Reply:
You can keep this dependency `gomodules.xyz/jsonpatch/v2`. We already import it transitively, because `sigs.k8s.io/controller-runtime` uses it.